src/scanners/css.js

```diff
@@ -12,7 +12,7 @@ export default class CSSScanner extends BaseScanner {
 
   _defaultRules = rules;
 
-  processCode(cssCode, cssInstruction) {
+  processCode(cssCode, cssInstruction, rules) {
```

Comment:
Is this for injection? Might be better to do `_rules=rules` if that's the case to make the distinction.